app/src/features/apiClient/helpers/modules/scriptsV2/worker/script-internals/types.ts

```diff
@@ -19,11 +19,18 @@ interface StatusAssertions {
 }
 
 export type LocalScopeRequest = RQAPI.Request & {
-  toJSON: () => {
-    method: string;
-    url: string;
-    body: object;
-  };
+  toJSON: () =>
+    | {
+        method: string;
+        url: string;
+        body: object;
+      }
+    | {
+        url: string;
+        operation: string;
+        variables: object;
+        operationName?: string;
+      };
```

Comment:
Not sure if this will be the interface. Currently, scripts run after converting the request to HTTP